posts/how-kubernetes-add-to-your-business.md

```diff
@@ -0,0 +1,60 @@
+---
+authors:
+- Jeet Gajjar
+tags:
+- kubernetes
+- docker
+- microservices
+
+date: 
+title: "How Kubernetes Adds to Your Business"
```

Comment:
I'm thinking something more like "How Kubernetes Can Benefit Your Business Needs"

Reply:
Sure, I like that. Making the change now